cloud/types.go

```diff
@@ -60,7 +60,8 @@ type LBSpec struct {
 
 // RouteTableSpec defines the specification for a route table
 type RouteTableSpec struct {
-	Name string
+	Name           string
+	IsControlPlane bool
```

Comment:
```suggestion
	MachineRole string
```

https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/816/files#diff-e0a02b9b279470c6d5411301fba575a9L33

Reply:
ok, similar to the Subnet I guess